src/game.js

```diff
@@ -70,76 +70,11 @@ class Game {
     }
   }
 
-  evaluateTopRow() {
+  evaluateForThree(boxOne, boxTwo, boxThree) {
```

Comment:
Great job with this solution!! I think this is a really strong refactor that really DRYs up this method.